package/.eslintrc.cjs

```diff
@@ -0,0 +1,61 @@
+// eslint-disable-next-line no-undef
+module.exports = {
+    env: {
+        browser: true,
+        es2021: true
+    },
+    extends: [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    overrides: [
+    ],
+    parser: "@typescript-eslint/parser",
+    parserOptions: {
+        ecmaVersion: "latest",
+        sourceType: "module"
+    },
+    plugins: [
+        "@typescript-eslint"
+    ],
+    ignorePatterns: [
+        "**/*.js",
+        "**/*.d.ts"
+    ],
+    rules: {
+        indent: [
+            "error",
+            4
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        quotes: [
+            "error",
+            "double"
+        ],
+        semi: "off",
+        "@typescript-eslint/semi": ["error", "never"],
+        "@typescript-eslint/member-delimiter-style": [
+            "error",
+            {
+                multiline: {
+                    delimiter: "none",
+                    requireLast: false
+                },
+                singleline: {
+                    delimiter: "comma",
+                    requireLast: false
+                }
+            }
+        ],
+        "quote-props": [
+            "warn",
+            "as-needed"
+        ],
+        "@typescript-eslint/no-explicit-any": [
+            "off"
```

Comment:
Do we need `any`` somewhere in the code?

Reply:
```ts
const mapsReq = (await axios({ url: "https://valorant-api.com/v1/maps" }).catch(() => null))?.data?.data
if (mapsReq) {
  this.mapImages = {} as any
  mapsReq.forEach((map: any) => (this.mapImages[map.displayName] = {
    splash: map.splash,
    minimap: map.displayIcon,
    landscape: map.listViewIcon
  }))
}```
This is just a small example, as ts throws an error because `{}` is technically not the mapImages type, but there are parts where Ill change it

Reply:
Ok, yeah I think we can allow any for now.

Reply:
But you could also do explicit typing here, or not?